File: src/main/java/org/keynote/godtools/android/api/TranslationsApi.java
package org.keynote.godtools.android.api;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Streaming;

public interface TranslationsApi {
    String PATH_TRANSLATIONS = "translations";

    @Streaming
    @GET(PATH_TRANSLATIONS + "/{id}")
    Call<ResponseBody> download(@Path("id") long id);
}

File: library/api/src/main/kotlin/org/cru/godtools/api/TranslationsApi.kt
package org.cru.godtools.api

import okhttp3.ResponseBody
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Streaming

private const val PATH_TRANSLATIONS = "translations"

interface TranslationsApi {
    @Streaming
    @GET("$PATH_TRANSLATIONS/{id}")
    suspend fun download(@Path("id") id: Long): Response<ResponseBody>
}
